libs/dh/core/shell/src/lib/dh-core-shell.component.html

```diff
@@ -30,6 +30,16 @@
   </ng-container>
 
   <ng-container watt-shell-toolbar>
+    @if (breadcrumbService.breadcrumbs().length > 0) {
+      <watt-breadcrumbs>
+        @for (breadcrumb of breadcrumbService.breadcrumbs(); track breadcrumb.label) {
+          <watt-breadcrumb [routerLink]="breadcrumb.url">
+            {{ breadcrumb.label | transloco }}
```

Comment:
issue: this produces a console error. Looks like the translation is not available when the app bootstraps then becomes available at some point. The MP ID should not be translated. So the pipe here should probably be dropped, or somehow figure out if the `label` needs to be translated.

![image](https://github.com/user-attachments/assets/444d7051-8bb6-4936-991c-bd3f8dbdddd7)